retrofit/src/test/java/retrofit/EndpointsTest.java

```diff
@@ -10,4 +10,20 @@
     Endpoint endpoint = Endpoint.createFixed("http://example.com");
     assertThat(endpoint.url()).isEqualTo("http://example.com");
   }
+
+  @Test public void emptyUrlThrows() {
+    try {
+      Endpoint.createFixed(null);
+    } catch (NullPointerException expected) {
```

Comment:
Normally we name these just `e` when the type is used. `expected` is for when we expect it but don't want to waste the comment in the empty `catch` block.